packages/ai-core/README.md

```diff
@@ -0,0 +1,166 @@
+# @sap-ai-sdk/ai-core
+
+This package provides tools to manage your scenarios and workflows in SAP AI Core.
+
+Streamline data preprocessing and model training pipelines
+Execute batch inference jobs
+Deploy inference endpoints for your trained models
+Register custom Docker registries, sync AI content from your own git repositories, and register your own object storage for training data and model artifacts
+
+### Installation
+
+```
+$ npm install @sap-ai-sdk/ai-core
+```
+
+## Pre-requisites
+
+- [Enable the AI Core service in BTP](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/initial-setup)
+- Project configured with Node.js v20 or higher and native ESM support enabled
+- For testing your application locally:
+  - Download a service key for your AI Core service instance
+  - Create a `.env` file in the sample-code directory
+  - Add an entry `AICORE_SERVICE_KEY='<content-of-service-key>'`
+
+## List of Available APIs
+
+We maintain a list of [currently available and tested AI Core APIs](https://github.com/SAP/ai-sdk-js/blob/main/docs/list-tested-APIs.md)
+
+## Usage
```

Comment:
In the technical writing course, I believe they taught us to avoid chaining headers without text like
```
# With
## This
### Example
```
Not sure what content we would put here instead, but if something comes to your mind consider adding it.